core/src/luigi-config/navigation/static-navigation-model.js

```diff
@@ -1544,6 +1544,53 @@ export function getStaticRootNodes(
         },
       ],
     },
+    // STORAGE CATEGORY (CLUSTER)
+    {
+      category: {
+        label: i18next.t('storage.title'),
+        icon: 'sap-box',
+        collapsible: true,
+      },
+      pathSegment: '_storage_category_placeholder_',
+      hideFromNav: true,
+    },
+
+    {
+      category: i18next.t('storage.title'),
+      resourceType: 'persistentvolumes',
+      pathSegment: 'persistentvolumes',
+      label: i18next.t('pv.title'),
+      viewUrl:
+        config.coreUIModuleUrl +
+        '/persistentVolumes?' +
+        toSearchParamsString({
+          resourceApiPath: '/api/v1',
+          hasDetailsView: true,
+        }),
+      viewGroup: coreUIViewGroupName,
+      keepSelectedForChildren: true,
+      context: {
+        requiredFeatures: [features.ISTIO],
```

Comment:
Is Istio required for PVs? I thought it's a standard k8s resource, at least from docs and seeing `api/` instead of `apis/`.